frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -390,6 +400,57 @@ function ReaderHighlight:addToMainMenu(menu_items)
             separator = i == #highlight_style,
         })
     end
+    table.insert(menu_items.highlight_options.sub_item_table, {
```

Comment:
Let's hide it for b/w devices.

Reply:
It could still be used to change the drawer on b/w devices, but I'm not sure how useful that is.
For now they have lived fine without the button, so I'll hide it like you said.